netket/utils/symmetry/axial.py

```diff
@@ -0,0 +1,225 @@
+# Copyright 2021 The NetKet Authors - All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .semigroup import Identity
+from .point_group import PGSymmetry, PointGroup
+from netket.utils.types import Array
+import numpy as np
+
+
+def _rotation(angle: float, axis: Array) -> PGSymmetry:
+    """Returns a rotation by `angle` degrees around `axis`."""
+    angle = np.radians(angle)
+    axis = np.asarray(axis) / np.linalg.norm(axis)
+    return PGSymmetry(
+        np.cos(angle) * np.eye(3)
+        + np.sin(angle) * np.cross(np.eye(3), axis)
+        + (1 - np.cos(angle)) * np.outer(axis, axis)
+    )
+
+
+def C(n: int, axis: Array = np.array([0, 0, 1])) -> PointGroup:
```

Comment:
I'd rather convert to array inside and accept anything (tuple, list, array...)

Reply:
I forgot if it breaks from non-arrays, but I'll make sure it doesn't

Reply:
`np.asarray` is almost free for stuff that are already arrays so you can just slap it in front.

Reply:
I know, I just don't remember if I already did it or not